external-import/zerofox/src/requirements.txt

```diff
@@ -1,4 +1,8 @@
+requests==2.31.0
```

Comment:
on master `pycti` version will be 6.4.2 which will raise an error during `pip install`: 
![image](https://github.com/user-attachments/assets/f602a979-ecd1-4906-9d9b-d0f73bd566d5)

I encourage you to upgrade `requests` lib

Reply:
Hi @Powlinett  thank you for the suggestion,
I loosened up the dependencies

regarding the config.yml, is there documentation on how to include that setup?

Reply:
Thanks 🙂

You can find some documentation about `config.yml` [here](https://docs.opencti.io/latest/deployment/connectors/#parameters) and [an example](https://github.com/OpenCTI-Platform/connectors/blob/master/external-import/tenable-vuln-management/src/tenable_vuln_management/config_variables.py) of the way we handle config from both environment variables _and_ `config.yml` file.
It's useful if you want to [manually launch a connector](https://docs.opencti.io/latest/deployment/connectors/#manual-activation), for dev or debug purpose for instance.

But it's entirely up to you, we can still merge this PR without any further changes now, the intent of my comment was just to be sure that our team was aware of this "limitation".

Let us know 😇